pmpsui/preemptive_requests.py

```diff
@@ -453,6 +478,15 @@ def update_jf_trans_for_label(self, label: QtWidgets.QLabel) -> None:
             scaled = raw_trans
         label.setText(f"{scaled:.2e}")
 
+    def update_veto(self, value, byte):
```

Comment:
It'd be nice to hint the type of `value` (really the whole signature) here; are we expecting an array of ints?  For those of us (me) who aren't as familiar with the veto pv waveforms.